perimeterx-nginx-plugin-2.10-1.rockspec

```diff
@@ -1,24 +0,0 @@
- package = "perimeterx-nginx-plugin"
```

Comment:
I assume the deletion here is because of rename, but you probably forgot to `git add` the renamed file